src/main/java/org/damap/base/conversion/AbstractTemplateExportSetup.java

```diff
@@ -53,21 +53,31 @@ protected void exportSetup(long dmpId) {
     costList = dmp.getCosts();
 
     // determine project leader/coordinator/principal investigator
-    Optional<Contributor> projectLeaderOpt =
+    projectCoordinators =
         dmp.getContributorList().stream()
             .filter(
-                contributor -> contributor.getContributorRole() == EContributorRole.PROJECT_LEADER)
-            .findFirst();
-    if (projectLeaderOpt.isPresent())
-      projectCoordinator =
-          ContributorDOMapper.mapEntityToDO(projectLeaderOpt.get(), new ContributorDO());
-    else
+                contributor ->
+                    contributor.getContributorRole() == EContributorRole.PROJECT_LEADER
+                        || contributor.getContributorRole() == EContributorRole.PROJECT_COORDINATOR
+                        || contributor.getContributorRole()
+                            == EContributorRole.PRINCIPAL_INVESTIGATOR)
+            .map(contributor -> ContributorDOMapper.mapEntityToDO(contributor, new ContributorDO()))
+            .toList();
+
+    if (projectCoordinators.isEmpty()) {
       try {
-        if (dmp.getProject() != null && dmp.getProject().getUniversityId() != null)
-          projectCoordinator = projectService.getProjectLeader(dmp.getProject().getUniversityId());
+        if (dmp.getProject() != null && dmp.getProject().getUniversityId() != null) {
+          ContributorDO projectLeader =
+              projectService.getProjectLeader(dmp.getProject().getUniversityId());
+          if (projectLeader.getRole() == null) {
+            projectLeader.setRole(EContributorRole.PROJECT_LEADER);
+          }
+          projectCoordinators = List.of(projectLeader);
+        } else projectCoordinators = new ArrayList<>();
```

Comment:
This can be slightly refactored, since projectCoordinators is already a list at that point.